### PR TITLE
LPS-137227 Test Autom: Editor permission for Page Audit

### PR DESCRIPTION
**Pull request by Luizcarlosqueiroz**
**JIRA Ticket:**
LPS-137227
LPS-131668

**Motivation:**
Test Automation for "As a content management, I want to see the Page Audit panel if I have permissions to edit the page"

### CODE REVIEW

**Comment by Luizcarlosqueiroz** [comment]
@CarlosBrichete

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/pageaudit/PageAudit.testcase [hunk 1]
```diff
@@ -555,6 +555,2087 @@ definition {
...
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+		}
+
+		task ("Remove user and role created") {
+			JSONUser.tearDownNonAdminUsers();
+
+			JSONRole.deleteRole(roleTitle = "BlogsEditorRole");
+		}
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/pageaudit/PageAudit.testcase:650]: This should be included in the teardown, not in the test.
For the line _JSONUser.tearDownNonAdminUsers();_, you can just included it in the teardown.
For the line **JSONRole.deleteRole(roleTitle = "BlogsEditorRole");**, if you add the line _Role.tearDownCP();_ in the teardown, all roles with the word "Name" in the name of the role will be deleted, so I would include this line in the teardown and modify all the role names, to include the word "Name" :)

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]

**Comment by CarlosBrichete** [comment]
Hey @Luizcarlosqueiroz , please modify the tests with the review comment, thank you :)

**Comment by Luizcarlosqueiroz** [comment]
Hey @CarlosBrichete! Tests modified.

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by ling-alan-huang** [comment]
ci:test:sf

**Comment by ling-alan-huang** [comment]
Sent SF fix: https://github.com/hhuijser/liferay-portal/pull/5838

**Comment by Luizcarlosqueiroz** [comment]
ci:test:sf

**Comment by Luizcarlosqueiroz** [comment]
ci:test:sf

**Comment by Luizcarlosqueiroz** [comment]
@CarlosBrichete Just to leave a record of it, as seen in ci:test:sf, `CheckNotVisibleReloadbButtonInPrivatePage` test was duplicated. I removed one of them.

**Comment by Luizcarlosqueiroz** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
ci:forward

**Pull request closed without merging**